addons/infinite-scroll/addon.json

```diff
@@ -7,6 +7,11 @@
     },
     {
       "name": "Hans5958"
+    },
+    {
+      "name": "Jazza",
+      "note": "Explore page",
```

Comment:
I'd suggest you remove this note. I imagine you added it to avoid getting "too much credit" for doing a relatively small change. But it's not necessary.

Reply:
Hah yeah I didn't want too much credit, perhaps I just remove myself from it all together?

Reply:
> Hah yeah I didn't want too much credit, perhaps I just remove myself from it all together?

It's an understandable feeling, but do add yourself like normal. You're contributing, and you deserve credit. It's understood that those who contributed may have contributed different amounts, but they all deserve credit just the same.

Reply:
Alright, thanks for your view on the matter BroJac, I'll keep myself in the credits without a note as it is now and resolve this conversation :D

Reply:
By "it" I meant the note. You should keep your credit object.